frontend/sw360-portlet/src/main/webapp/css/licenses.css

```diff
@@ -118,4 +118,10 @@ tr {
 		margin-right:90px;
 		cursor: pointer; cursor: hand; 
 		display:inline;
-}
\ No newline at end of file
+}
+#tab-LicenseText{
+	overflow: hidden;
```

Comment:
Is that your solution to remove the scroll-bar? Together with https://github.com/siemens/sw360portal/pull/94/files#diff-972eef148286086787d19cde5b1b0038R32?

I am not sure whether hiding the overflow is a good idea here. Can you demonstrate it to me tomorrow and show, where the styling is used?